### PR TITLE
Implementation of modulo line numbers

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`take row 15` => will select 15/115/215/...

### CODE REVIEW

File: src/marks/lines_number.py [hunk 1]
```diff
@@ -1,74 +1,45 @@
...
-        "lineNumber": direction["transformation"](line_number),
-        "isRelative": direction["isRelative"],
-    }
-
+# NOTE: Please do not change these dicts.  Use the CSVs for customization.
+# See https://github.com/pokey/cursorless-talon/blob/main/docs/customization.md
+directions = [
+    CustomizableTerm("row", "modulo100", "modulo100", lambda number: number - 1),
+    CustomizableTerm("up", "relativeUp", "relative", lambda number: -number),
+    CustomizableTerm("down", "relativeDown", "relative", lambda number: number),
```
Comment (pokey) [inline src/marks/lines_number.py:23]: Given that these are lumped into same csv with other special marks, maybe make identifier a bit more verbose?

```suggestion
    CustomizableTerm("row", "modulo100", "lineNumberModulo100", lambda number: number - 1),
    CustomizableTerm("up", "lineNumberRelativeUp", "relative", lambda number: -number),
    CustomizableTerm("down", "lineNumberRelativeDown", "relative", lambda number: number),
```

File: src/marks/mark.py [hunk 2]
```diff
@@ -84,9 +85,7 @@ class CustomizableTerm:
     rule=(
         "<user.cursorless_decorated_symbol> | "
         "{user.cursorless_special_mark} |"
-        # Because of problems with performance we have to have a simple version for now
-        # "<user.cursorless_line_number>" # row, up, down
-        "<user.cursorless_line_number_simple>"  # up, down
+        "<user.cursorless_line_number>"  # row, up, down
```
Comment (pokey) [inline src/marks/mark.py:88]:
```suggestion
        "<user.cursorless_line_number>"  # row mod 100, up, down
```

**Review by pokey — APPROVED** [review]
This is really nice. I left a couple minor comments but otherwise looks great

File: src/marks/mark.py (hunk 2 above)
Comment (AndreasArvidsson) [reply src/marks/mark.py:88]: The comment here is made more as a reference to the spoken phrase and not the actual implementation

File: src/marks/lines_number.py (hunk 1 above)
Comment (AndreasArvidsson) [reply src/marks/lines_number.py:23]: Agreed

File: src/marks/mark.py (hunk 2 above)
Comment (pokey) [reply src/marks/mark.py:88]: Hmm ok.  Just wanted to have some indication somewhere that we're currently only supporting modulo.  Maybe the following?

```suggestion
        "<user.cursorless_line_number>"  # row (ie absolute mod 100), up, down
```

Comment (AndreasArvidsson) [reply src/marks/mark.py]: Works for me

**Review by pokey — APPROVED** [review]

**Pull request merged**